astroquery/gaia/core.py

```diff
@@ -595,5 +835,95 @@ def cross_match(self, full_qualified_table_name_a=None,
                                      upload_resource=None,
                                      upload_table_name=None)
 
+    def launch_job(self, query, name=None, output_file=None,
```

Comment:
this may be an excellent time to start using kwarg only arguments for the optional kwargs, that will provide easier maintenance down the line